static/app/components/feedback/feedbackOnboarding/sidebar.tsx

```diff
@@ -192,32 +192,30 @@ function OnboardingContent({currentProject}: {currentProject: Project}) {
     replayJsLoaderInstructionsPlatformList.includes(currentPlatform.id) &&
     !crashReportOnboarding;
 
-  function getJsFramework() {
-    return (
-      replayJsFrameworkOptions.find(p => p.id === jsFramework.value) ??
-      replayJsFrameworkOptions[0]
-    );
-  }
+  const jsFrameworkPlatform =
+    replayJsFrameworkOptions.find(p => p.id === jsFramework.value) ??
+    replayJsFrameworkOptions[0];
 
   const {
+    isLoading: isProjKeysLoading,
```

Comment:
```suggestion
    isLoading,
```

can probably just change the use of `isProjKeysLoading` down below to avoid the alias here - it's only referenced once :)